Documentation/user-guides/getting-started.md

```diff
@@ -404,7 +404,7 @@ Further reading:
 
 
 [alerting-guide]: alerting.md
-[cluster-monitoring]: cluster-monitoring.md
+[cluster-monitoring]: ?????
```

Comment:
@metalmatze Sorry, don't have context what should be here instead..  :thinking: any ideas?

Reply:
I have no idea what this was meant, pre my time :) cc @brancz

Reply:
there used to be cluster-monitoring docs, but those pointed to kube-prometheus, I believe we removed it as it always tended to be out of date compared to kube-prometheus. If we want to we can just link out to kube-prometheus